website_sale_search_fuzzy/hooks.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, SUPERUSER_ID
+from odoo.exceptions import UserError
+
+
+def _add_trgm_index_product_tmpl_name(cr, registry):
+    with cr.savepoint():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        trgm_mod = env['trgm.index']
+
+        if trgm_mod._trgm_extension_exists() != 'installed':
+            raise UserError(_(
```

Comment:
The function is used in base_search_fuzzy, however it seems the logic in the module only logs warnings if it isn't installed.

Seems like some tests are cancelled out if it isn't installed either.
https://github.com/OCA/server-tools/search?utf8=%E2%9C%93&q=_trgm_extension_exists&type=